tests/sqlparser_bigquery.rs

```diff
@@ -50,112 +50,146 @@ fn parse_literal_string() {
     let select = dialect.verified_only_select(sql);
     assert_eq!(12, select.projection.len());
     assert_eq!(
-        &Expr::Value(Value::SingleQuotedString("single".into())),
+        &Expr::Value(
+            Value::SingleQuotedString("single".into())
+                .with_span(Span::new(Location::new(1, 7), Location::new(1, 12)))
+        ),
```

Comment:
Perhaps there's an automated way to do this, instead of manually writing locations?

I had to do something similar in one of my DataFusion PRs:

https://github.com/eliaperantoni/datafusion/blob/7265bd2d4d6fd86d34ce69672739b10aa4995b8c/datafusion/sql/tests/cases/diagnostic.rs#L132-L140

Reply:
Yes, that would tremendously reduce the amount of work to do in the tests!